contracts/ClaimIssuer.sol

```diff
@@ -79,15 +82,8 @@ contract ClaimIssuer is IClaimIssuer, Identity, UUPSUpgradeable {
             _uri
         );
 
-        (bool success, ) = address(_identity).call(
-            abi.encodeWithSelector(
-                _identity.execute.selector,
-                address(_identity),
-                0,
-                addClaimData
-            )
-        );
-        require(success, Errors.CallFailed());
+        _identity.execute(address(_identity), 0, addClaimData);
```

Comment:
can make a try/catch here to revert with a custom error in case the target contract is not an identity and it reverts